### PR TITLE
Billing Installer

### PR DESCRIPTION
**Pull request by sadhasivam-ladybirdweb**
### Issue: #3103

Change the billing installer to be similar to Faveo.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -57,24 +57,24 @@
...
-
-    Route::get('environment/wizard', [Installer\EnvironmentController::class, 'environmentWizard'])->name('environmentWizard');
-
-    Route::post('environment/saveWizard', [Installer\EnvironmentController::class, 'saveWizard'])->name('environmentSaveWizard');
-
-    Route::get('database', [Installer\DatabaseController::class, 'database'])->name('database');
-
-    Route::get('final', [Installer\FinalController::class, 'finish'])->name('final');
-});
+//
```
Comment (sandesh556) [inline routes/web.php:60]: Lets remove all of this if not needed

File: app/Http/Controllers/BillingInstaller/BillingDependencyController.php [hunk 2]
```diff
@@ -0,0 +1,383 @@
+<?php
```
Comment (sandesh556) [inline app/Http/Controllers/BillingInstaller/BillingDependencyController.php:1]: Make sure all messages are coming from lang

**Review by sandesh556 — CHANGES_REQUESTED** [review]

**Review by sandesh556 — DISMISSED** [review]

**Review by Sakthi002 — APPROVED** [review]

**Review by laksh-22 — APPROVED** [review]

**Review by RafficMohammed — APPROVED** [review]

**Comment by Sabaric12** [comment]
Round 1 Testing Summary:
Tested this pr the issue sheet is mentioned below.
Total time in days: 5 hours
Total issues: 8
Minor - 8 
Major - 0 
Critical - 0
No of times PR was sent for Functionality correction:1
PR breaks existing functionality: No

**Comment by indira15v** [comment]
**Round two testing summary**
Tested this PR in local 
Issue sheet
https://docs.google.com/spreadsheets/d/17WwJ7Zfbo5DcOK41CyWH_GE2xkzAGM-H2kB50pxwKJ8/edit?gid=0#gid=0
Total issues: 5
Minor - 1
Major - 4
Critical - 0
No of times PR was sent for Functionality correction:1
PR breaks existing functionality: No

**Comment by sadhasivam-ladybirdweb** [comment]
CLI installation has also been implemented in this PR, so please review it in both the UI and the CLI.

**Comment by indira15v** [comment]
Retesting summary
Retested this PR and facing issues mentioned in sheet
https://docs.google.com/spreadsheets/d/17WwJ7Zfbo5DcOK41CyWH_GE2xkzAGM-H2kB50pxwKJ8/edit?gid=0#gid=0

**Comment by Sabaric12** [comment]
Round 1 Testing Summary:
Tested this pr the issue sheet is mentioned below.
Total time in days: 2 hours
Total issues: 2
Minor - 1
Major - 1
Critical - 0
No of times PR was sent for Functionality correction:2
PR breaks existing functionality: No
https://docs.google.com/spreadsheets/d/17WwJ7Zfbo5DcOK41CyWH_GE2xkzAGM-H2kB50pxwKJ8/edit?gid=0#gid=0

**Comment by Sabaric12** [comment]
Round 1 Testing Summary:
Tested this pr with the fresh installation in the Local and the Server with the Zip File in the GUI and the CLI mentioned issues are working fine now.
Total time in days: 2 hours
Total issues: 11
Minor - 7
Major - 4
Critical - 0
No of times PR was sent for Functionality correction:2
PR breaks existing functionality: No
Note:
**https://docs.google.com/spreadsheets/d/17WwJ7Zfbo5DcOK41CyWH_GE2xkzAGM-H2kB50pxwKJ8/edit?gid=0#gid=0**

**Comment by indira15v** [comment]
**Round two retesting summary**
Tested this PR  with the fresh installation in the the Server with the Zip File, the GUI and the CLI method also working fine 
No of times PR  sent for Functionality correction:2
PR breaks existing functionality: No
Issue sheet 
https://docs.google.com/spreadsheets/d/17WwJ7Zfbo5DcOK41CyWH_GE2xkzAGM-H2kB50pxwKJ8/edit?gid=0#gid=0

**Review by indira15v — APPROVED** [review]

**Pull request merged**